src/prefab.ts

```diff
@@ -91,11 +99,19 @@ export const prefab = {
   },
 
   isEnabled(key: string): boolean {
-    return this.get(key) === true;
+    const value = this.get(key) === true;
+
+    this.trackEvaluation(key, value, this.context);
+
+    return value;
   },
 
   get(key: string): ConfigValue {
-    return this.configs[key]?.value;
+    const value = this.configs[key]?.value;
+
+    this.trackEvaluation(key, value, this.context);
```

Comment:
Yeah I was wondering whether or not we needed something like that. I originally skipped because I think the tracking clients we're typically calling here do their work async. But it seems fine to add as safety.

My only question would be about event sequencing. Are there any guarantees about the ordering of execution when you use setTimeout like this?

Reply:
Sequencing is interesting. I don't think it'll matter in practice, but we can change the behavior if we experience places where it does.